assets/sass/components/_alerts.scss

```diff
@@ -0,0 +1,19 @@
+
+.error-summary-outer {
+	width: 100%;
+	background: repeating-linear-gradient(
+		45deg,
+		#a94442,
```

Comment:
Thinking in terms of "magic numbers", could these be replaced with a variable at the top of the file?

Reply:
This was something I was meant to ask you about now that Sass has been added. Where would be the correct location for an "error-color"? And what's the correct way to declare it? The red color defined there is the same one, bs3 uses for alert texts, that is why I chose it. And then again, the white color there is the general background color of A+.

Reply:
Good question.

Our SASS migration isn't finished. In PR #645 I've moved to new universal colours. If that is merged first, I'd use `$aplus-danger` as the red, `$aplus-background` as white. Ideally no components will include colour definitions.

Another option is to use the Bootstrap-specific variables. This works, but I don't want to do this in the longer term, when we have custom variables with more consistent use.

If this needs to be merged time-sensitively, then it might be easiest to just move these to variables at the top of the file, then replace them later. 

> the white color there is the general background color of A+

For now. we might want a "dark mode" one day, so I'd still recommend reading this from a variable.

Reply:
I also removed the use of magic numbers, and used the same shade of red you used with #645. If your button-thing is merged before, I can then move these. If not, at least there is now the continuity with colors.

Reply:
This is good, now we just have to remember to remove these when we can!